nrich-search/README.md

```diff
@@ -4,17 +4,22 @@
 
 ## Overview
 
-`nrich-search` is a module whose purpose is to make querying of JPA entities easier. Motivation for its creation were complex search forms on multiple projects. It's built on top of Spring Data JPA
-library and takes care of query composition from classes holding property values and additional `SearchConfigration` that defines how those property values should be applied (i.e. what operator should
-be used), what classes should be returned etc.
+`nrich-search` is a module whose purpose is to make querying of JPA entities easier.
+Motivation for its creation were complex search forms on multiple projects.
+It's built on top of the Spring Data JPA library and takes care of query composition from classes that hold query properties and from
+`SearchConfigration` class that defines how those query property values should be applied (i.e. what operator should be used),
+what classes should be returned etc.
 
 ## Setting up Spring beans
 
-To be able to use this module JPA repositories should be enabled with custom factory by placing `@EnableJpaRepositories(repositoryFactoryBeanClass = SearchExecutorJpaRepositoryFactoryBean.class)`
-annotation on `@Configuration` class and following beans should be defined in context:
+To be able to use this module, JPA repositories should be enabled with `SearchExecutorJpaRepositoryFactoryBean` set as repository factory bean class,
```

Comment:
I'm not sure if the reorganization of this particular section improves readability. I think it's pretty clear from the annotation definition that `SearchExecutorJpaRepositoryFactoryBean` should be set as repository factory bean -> `repositoryFactoryBeanClass = SearchExecutorJpaRepositoryFactoryBean.class`.

Reply:
I changed the formulation so it combines both. I think its better now.